src/components/Badge.html

```diff
@@ -57,33 +57,33 @@ <h3 class="h3Tag">Basic Badge And Color Badge</h3>
 
   <div class="BadgeContainerTwo paddingOtherContainer">
     <h3 class="h3Tag">With Badge</h3>
-    <p class="descriptionOne">Badge create Status of Badge</p>
+    <p class="descriptionOne discriptionText">Badge create Status of Badge</p>
 
     <div class="DOMContainer flex-row-center">
     
     <div class="AvatarDomMainContainer">
-        <div class="AvatarDomContainer descriptionOne statusAvatarFlexRightTop">
+        <div class="descriptionOne statusBadgeFlexRightTop">
         <img class="AvatarImage smallAvatar box-shadow-round" src="../Images/steve.jpg" alt="" sizes="" srcset="">
```

Comment:
Please remove the undesired attributes form img and also please add alt to the image tag